packages/block-editor/src/components/block-list/style.scss

```diff
@@ -649,6 +649,9 @@
 
 .block-editor-block-list__layout.is-root-container {
 	// Full-wide. (to account for the padddings added above)
+	// The first two rules account for the alignment wrapper div for the image block.
+	> div > .block-editor-block-list__block[data-align="full"],
+	> div > .block-editor-block-list__block.alignfull,
```

Comment:
maybe we can add `> div:not(.block-editor-block-list__block)` instead of  `> div` because that markup is definitely possible in nested contexts

Reply:
I suppose we're already in "very very specific selectors" territory — but the :not rule would make it even more specific. But I'm happy to add it.